andela/bp-esa-backend
# #163032534 Send email if fetching from allocations fails for a number of times

#### What does this PR do?
Restarts the automation if fetching placement data fails. The restart occurs for a number of times and if the fails for a number of times, an email is sent to support

#### Description of Task to be completed?
- Create env variable to be the number of times to restart the automation if fetching fails
- Update the .env.example file
- Create a method to send email if fetching fails

#### How should this be manually tested?
- Git clone the repository
- Checkout to `ft-restart-automation-if-fetching-placement-fails-163032279`
- Install necessary dependencies
- Ensure the .env file is updated with the required variables
- Go to the allocations.js file and change to allocations url to ensure it fails
- Run `npm start`

#### Any background context you want to provide?
- In the env files, `SUPPORT_EMAIL=` Should be your email so that you can test the feature

#### What are the relevant pivotal tracker stories?
[#163032279](https://www.pivotaltracker.com/story/show/#163032279)
[##163032534](https://www.pivotaltracker.com/story/show/##163032534)

#### Postman Documentation
N/A

#### Screenshots (If applicable)
N/A
